media/src/project-activity-components/layers/layer.tsx

```diff
@@ -36,6 +37,10 @@ export const Layer: React.FC<LayerProps> = (
         setUpdatedLayerTitle(layer.title);
     }, [layer.title]);
 
+    useEffect(() => {
+        setUpdatedLayerColor(layer.color);
+    }, [layer.color]);
+
```

Comment:
This can probably be moved in with the useEffect in line 36 since they are related

Reply:
Ugh, re-basing is so frustrating sometimes. Thanks. I took care of it.